templates/aws/nodes/terraform/pools/main.tf

```diff
@@ -0,0 +1,70 @@
+terraform {
+  required_version = ">= 0.13.1"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "random" {}
+provider "aws" {
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+  region =  var.aws_region
+}
+
+resource "random_id" "cluster_id" {
+  byte_length       = 6
+}
+
+resource "aws_key_pair" "corral_key" {
+  key_name       = "corral-${var.corral_user_id}-${random_id.cluster_id.hex}"
+  public_key = var.corral_public_key
+}
+
+resource "aws_instance" "node" {
+  count = var.node_count
+  ami = var.aws_ami
+  instance_type     = var.instance_type
+  key_name = aws_key_pair.corral_key.key_name
+  vpc_security_group_ids = [var.aws_security_group]
+  subnet_id = var.aws_subnet
+  associate_public_ip_address = var.airgap_setup ? false : true
+
+  ebs_block_device {
+     device_name           = "/dev/sda1"
+     volume_size           = "200"
```

Comment:
https://github.com/rancherlabs/corral-packages/pull/12#discussion_r1214526780
@slickwarren I can do that. I'll make the update.

Reply:
I decided to make the volume_type a variable as well.